godoc/dirtrees.go

```diff
@@ -332,7 +332,7 @@ func hasThirdParty(list []DirEntry) bool {
 // If filter is set, only the directory entries whose paths match the filter
 // are included.
 //
-func (root *Directory) listing(skipRoot bool, filter func(string) bool) *DirList {
+func (root *Directory) listing(mode PageInfoMode, skipRoot bool, filter func(string) bool) *DirList {
```

Comment:
warning: receiver name root should be consistent with previous receiver name dir for Directory (golint)